src/org/traccar/database/ConnectionManager.java

```diff
@@ -49,6 +53,9 @@
     public ConnectionManager() {
         deviceTimeout = Context.getConfig().getLong("status.timeout", DEFAULT_TIMEOUT) * 1000;
         enableStatusEvents = Context.getConfig().getBoolean("event.enable");
+        if (Context.getConfig().getBoolean("status.updateDeviceState")) {
```

Comment:
I think it's OK. I'm assuming that the idea is that state will have more information than just motion state.